### PR TITLE
Handle newline in session abstracts for ELO vote

### PR DESCRIPTION
**Pull request by bplowry**
This is a bit selfish because my own submissions had two paragraphs and I noticed they weren't appearing

|Before|After|
|---|---|
|Screen Shot 2022-04-21 at 00 05 22|Screen Shot 2022-04-21 at 00 05 04|

### CODE REVIEW

File: components/Voting/Elo.tsx [hunk 1]
```diff
@@ -52,10 +53,17 @@ function EloChoice({ session, variant = 'primary', onChoice }: EloChoiceProps) {
...
+  const abstract = session.Abstract
+  const paragraphs = useMemo(() => abstract.split('\n').filter((para) => !!para), [abstract])
+
   return (
     <StyledEloChoice variant={variant}>
       <StyledSessionTitle>{session.Title}</StyledSessionTitle>
-      <StyledSessionAbstract>{session.Abstract}</StyledSessionAbstract>
+      <StyledSessionAbstract>
+        {paragraphs.map((para) => (
+          <p key={para}>{para}</p>
```
Comment (antonjb) [inline components/Voting/Elo.tsx:64]: Would this look ok using the <Text> component?

**Review by antonjb — APPROVED** [review]
Nice catch 👏

File: components/Voting/Elo.tsx (hunk 1 above)
Comment (bplowry) [reply components/Voting/Elo.tsx:64]: The `StyledSessionAbstract` component? I'd need to move the `overflowY: auto` stuff onto a parent, but could do

Comment (antonjb) [reply components/Voting/Elo.tsx:64]: That's weird, the `<Text>` part disappeared. I just meant instead of `<p>`

Comment (bplowry) [reply components/Voting/Elo.tsx:64]: I'll swap it over. Looks basically the same: margin between paragraphs is a little bigger (24px vs 8px), colour is slightly different (#292929 vs #333)

**Pull request merged**